packages/terra-status-view/src/StatusView.jsx

```diff
@@ -99,7 +100,7 @@ const StatusView = ({
   } else if (variant && !isGlyphHidden) {
     glyphSection = (
       <div className={cx('glyph')}>
-        <svg className={cx(variant)} />
+        <svg focusable={false} className={cx(variant)} />
```

Comment:
Does this need to be an SVG tag? Does switching to a div break/fix expected functionality? On initial glance this is just an empty container that is used to apply a CSS background image.

Reply:
On replacing svg with div, no image shows up.

Reply:
Do you know why no image shows up? We use divs/spans for background SVG images throughout the component library. The root issue for this bug is using the empty SVG container, if we need the SVG container then this fix works, but if we don't need the SVG container a more appropriate fix is likely swapping out the SVG tag for a div or span.

Reply:
@saket2403 when you replaced the svg with div, did you also update the style selector? those styles would affect how the markup is rendered: https://github.com/cerner/terra-core/blob/main/packages/terra-status-view/src/StatusView.module.scss#L49

Reply:
Updating the css selectors did work, replaced svg with div fe0e024